frontend/resourceadm/pages/ResourcePage/ResourcePage.tsx

```diff
@@ -38,20 +46,94 @@ export const ResourcePage = () => {
   useEffect(() => {
     if (repoStatus) {
       setHasMergeConflict(repoStatus.hasMergeConflict);
-      setIsLocalRepoInSync(
-        ((repoStatus.behindBy === 0 || repoStatus.behindBy === null) && repoStatus.aheadBy === 0) ||
-          repoStatus.aheadBy === null
-      );
     }
-  }, [repoStatus]);
+  }, [repoStatus, currentPage]);
```

Comment:
Question:
Isn't repoStatus updated when refetch is called?
And refetch is called whenever you navigate to a new page (and currentPage is updated) so then, do you need to use currentPage as a dependency here?

Reply:
Nope, you are correct. I added it when testing some features, but forgot to remove it. Great spot!